File: hackerRank/Multiples_of_3_&_5.java
// Problem Statement Link : https://www.hackerrank.com/contests/projecteuler/challenges/euler001/

import java.util.*;

public class Solution {

    public static long CalSum(int number, int divisor){

        long quotient = (long) number/divisor;

        long sum = divisor * ((quotient*(quotient+1))/2);

        return sum;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int testcase = in.nextInt();
        
        while(testcase>0) {                                 
            int number = in.nextInt();
            long ans = CalSum(number-1,3) + CalSum(number-1,5) - CalSum(number-1,15);
            System.out.println(ans);
            testcase--;
        }
    }
}

File: hackerRank/Multiples_of_3_&_5.py
# Problem Statement Link : https://www.hackerrank.com/contests/projecteuler/challenges/euler001/

#!/bin/python3

import sys

testcase = int(input().strip())

def CalSum(number, divisor):
    quotient = int(number / divisor)
    # print('quotient : ', quotient)
    summation = divisor * ((quotient*(quotient+1))/2)
    # print(summation)
    return int(summation)
    
for a in range(testcase):
    number = int(input().strip())
    ans = CalSum(number-1, 3) + CalSum(number-1, 5) - CalSum(number-1, 15)
    print(ans)



